### PR TITLE
Display LTS and Stable on the frontpage

### PR DESCRIPTION
**Pull request by phillipj**
Until v5.0.0 exists in https://nodejs.org/dist/index.json, only v4.x LTS will be displayed, in other words just as before besides some design changes:

lts screenshot

Will display two buttons side by side automagically, as soon as `index.json` contains a v5.0.0 release which is not marked as `lts`:

skjermbilde 2015-10-19 kl 10 52 47

Any thoughts?

Refs https://github.com/nodejs/new.nodejs.org/issues/200.

### CODE REVIEW

File: layouts/index.hbs [hunk 1]
```diff
@@ -17,14 +17,30 @@
...
                         {{{ project.banner.content }}}
                     </p>
-                {{else}}
-                      <p class="home-version">{{ labels.current-version }}: {{ project.currentVersion }}</p>
                 {{/if}}
 
-                <a href="https://nodejs.org/dist/{{ project.currentVersion }}/" id="home-downloadbutton" data-version="{{ project.currentVersion }}" data-dl-local="{{ labels.download-for }}">{{ labels.download }}</a>
+                <h2 id="home-downloadhead" data-dl-local="{{ labels.download-for }}">{{ labels.download }}</h2>
+
+                <a href="https://nodejs.org/dist/{{ project.currentVersions.lts }}/" class="home-downloadbutton" title="Download {{ project.currentVersions.lts }} LTS" data-version="{{ project.currentVersions.lts }}">
```
Comment (fhemberger) [inline layouts/index.hbs]: Texts need to be fully translatable, even "LTS" might need to be translated as well.

File: locale/en/index.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ labels:
   current-version: Current Version
   download: Download
   download-for: Download for
-  other-downloads: Other Downloads
+  other-downloads: Other LTS Downloads
```
Comment (fhemberger) [inline locale/en/index.md]: Should probably be two fields: "Other Downloads", "Other Stable/LTS Downloads"

File: layouts/index.hbs (hunk 1 above)
Comment (phillipj) [inline layouts/index.hbs]: :+1:

File: locale/en/index.md (hunk 2 above)
Comment (phillipj) [inline locale/en/index.md]: Just pushed an update fixing these issues.

**Comment by phillipj** [comment]
Just too clarify, this can be merged now rather than syncing with the release of v5.0.0. It was made that way intentionally as I noticed @rvagg commented (somewhere?) that whenever a new release got promoted to http://nodejs.org/dist, it automatically kicked of a rebuild of the website.

**Comment by bnb** [comment]
+1, a solid job and indicates that LTS is important, too.

**Comment by phillipj** [comment]
I think it would be valueable for us to track clicks on these links. Any chance I could access to the GA stats for nodejs.org @mikeal?

**Comment by jbergstroem** [comment]
I'd like to see a simple explanation of LTS if we're to introduce both buttons. "What should I download"

**Comment by phillipj** [comment]
@jbergstroem would a couple of keywords like shown in [#200 comment](https://github.com/nodejs/new.nodejs.org/issues/200#issuecomment-149282667) suffice?

**Comment by phillipj** [comment]
> I'd like to see a simple explanation of LTS if we're to introduce both buttons. "What should I download"

In response to @jbergstroem and other similar comments in #200, I just pushed an update with @rvagg suggested taglines.

#### v4.2.1 only (before v5 exists)

skjermbilde 2015-10-28 kl 11 53 47

#### v4.2.1 and v5

skjermbilde 2015-10-28 kl 11 53 06

**Comment by rvagg** [comment]
+1 let's queue this up, aiming for a v5 release tomorrow, if that slips then next Monday or Tuesday at the latest

**Comment by fhemberger** [comment]
Merged: e53ddb3.

I changed the wording from "Other LTS Downloads/Stable" to "Other Downloads: LTS/Stable".

**Comment by phillipj** [comment]
Nice @fhemberger :+1:

**Pull request closed without merging**